helm-org-ql.el

```diff
@@ -178,21 +178,27 @@ Is transformed into this query:
   "Return Helm source named NAME that searches BUFFERS-FILES with `helm-org-ql'."
   ;; Expansion of `helm-build-sync-source' macro.
   (helm-make-source name 'helm-source-sync
+    :filtered-candidate-transformer (lambda (candidates _)
+				      (let ((window-width (window-width (helm-window))))
```

Comment:
The function body isn't indented correctly.  I usually recommend using `aggressive-indent-mode` to ensure Elisp is always indented correctly.

Reply:
I do use `aggressive-indent-mode`, but apparently I also had to turn off `indent-tabs-mode`.